src/components/SideBar.js

```diff
@@ -0,0 +1,84 @@
+import React, { useState } from "react";
```

Comment:
Just for better user experience, you could add a button here which would clear all filters, because currently there is no way to clear the filters only when amending the URL.
:)